libtransmission/block-info.h

```diff
@@ -14,9 +14,7 @@ struct tr_block_info
 public:
     static auto constexpr BlockSize = uint32_t{ 1024U * 16U };
 
-    tr_block_info() noexcept
-    {
-    }
```

Comment:
Please revert this change. It took way too much efforts to fix the build last time: #6590

Reply:
Ah I see, we removed old mac support in https://github.com/transmission/transmission/pull/6911